### ISSUE TITLE
Salvage - Arrapago Remnants: Qutrub/Draugar stat cell droprate should be ZERO unless players have killed the required number of opposing mobs.

### ISSUE THREAD

**Issue by DeuceFFXI** [EdenServer/community#5829]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
[Salvage Data](https://docs.google.com/spreadsheets/d/1dQ3ExnPUJQzEeJ3z5iYojTkuGJ-D_CSCxrmXl1EgCc4/edit#gid=302535065)

As stated, neither Qutrubs or Draugars should drop any stat cells, i.e. Fractus x2, unless players have killed the required number of the other mobs.  I did a lot of testing on this, particularly on the first floor and have well over 400 relevant data points.

At no time did either a Draugar or Qutrub drop a single stat cell prior to me clearing the required number of opposing enemies.  For instance, no Draugars dropped stat cells, even when full killing all 24 multiple times, before killing any Qutrubs on the first floor.  In other runs, I killed multiple Qutrub, but not the 15 required to allow stat cells to drop.  When I moved to the Draugar side, again no stat cells would drop.

Additional evidence which supports this argument is that on the second floor in which players can never kill both Qutrub and Draugar on the same floor did either mob drop a stat cell.  Conversely, on the third floor where there are six Draugar and seven Qutrub it is possible for these to drop stat cells.  However, this only occurs after killing a certain number of the opposite mob, similar to the first floor, but the required number is lower.

On the third floor, most of my data collecting was done with killing either all of the Draugar or Qutrub first.  In those instances only the group that was cleared second dropped any stat cells.  The droprate seemed quite similar to the one on the first floor, ~15%.  I still need to figure out exactly how many of say Qutrubs need to be killed for Draugar to begin dropping stat cells on the third floor.  Evidently, because Draugar can drop them after all Qutrub have been killed on the third floor, the number is lower than the first floor's requirement of 15.

In summary,
1st Floor: Stat cells from Draugar only begin dropping after 15 Qutrub are killed and vice versa
2nd Floor: It is not possible to receive stat cells from either Draugar or Qutrub on this floor.
3rd Floor: Stat cells from Draugar only begin dropping after X Qutrub are killed and vice versa.

So, in short, the droprate should start at 0%, and increase to 15% after the requirements on each floor have been met.

Obviously, I'll need to do some additional testing on the third floor to determine what the required number is.  I'll reply to this bug report when I do.

**Comment by 59blargedy** [comment]
resolved in July 5th, 2024 patch
